### PR TITLE
Amend Ringbuffer, ReliableTopic and IQueue documentation

### PR DESCRIPTION
**Pull request by mmedenjak**
- format javadoc
- fix some copy-pasted and incomplete javadoc

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/ringbuffer/OverflowPolicy.java [hunk 1]
```diff
@@ -58,8 +66,8 @@
     /**
      * Gets the ID for the given OverflowPolicy.
      *
-     * This reason this ID is used instead of an the ordinal value is that the ordinal value is more prone to changes due to
-     * reordering.
+     * This reason this ID is used instead of an the ordinal value is that the
```
Comment (mustafaiman) [inline hazelcast/src/main/java/com/hazelcast/ringbuffer/OverflowPolicy.java]: `This reason` -> `The reason`

File: hazelcast/src/main/java/com/hazelcast/topic/ReliableMessageListener.java [hunk 2]
```diff
@@ -20,75 +20,91 @@
...
-     * If you intent to create a durable subscriber so you continue from where you stopped the previous
-     * time, load the previous sequence and add 1. If you don't add one, then you will be receiving the
-     * same message twice.
+     * Retrieves the initial sequence from which this ReliableMessageListener
+     * should start.
+     * <p>
+     * Return {@code -1} if there is no initial sequence and you want to start
+     * from the next published message.
+     * <p>
+     * If you intent to create a durable subscriber so you continue from where
```
Comment (mustafaiman) [inline hazelcast/src/main/java/com/hazelcast/topic/ReliableMessageListener.java]: `intent` is a noun. Should ne `intend`

**Comment by mmedenjak** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
Thank you for the reviews @taburet @mustafaiman !

**Pull request merged**